SYCL/Reduction/reduction_nd_conditional.cpp

```diff
@@ -3,6 +3,9 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
+//
+// Missing built-ins on AMD
```

Comment:
Is 'reduce' algorithm is missing only, or there is something else? If there is such info it would be useful to have in the comment.
Ok from me for SYCL/Reduction/reduction* tests.

Reply:
Specifically for this one it's `__spirv_GroupIAdd` that's missing, I've just pushed a patch specifying the exact missing built-ins where applicable.

Reply:
Thank you!